{"id":23015,"className":"java.util.concurrent.ExecutionException","methodName":"ExecutionException","signature":"\u003cjava.util.concurrent.ExecutionException: void ExecutionException(java.lang.Throwable)\u003e","javadoc":"/** \n * Constructs an \u003ctt\u003eExecutionException\u003c/tt\u003e with the specified cause. The detail message is set to: \u003cpre\u003e (cause \u003d\u003d null ? null : cause.toString())\u003c/pre\u003e (which typically contains the class and detail message of \u003ctt\u003ecause\u003c/tt\u003e).\n * @param cause the cause (which is saved for later retrieval by the{@link #getCause()} method)\n */\n","tf":{"freq":{"except":2,"detail":2,"save":1,"later":1,"messag":2,"link":1,"typic":1,"specifi":1,"caus":7,"retriev":1,"pre":2,"string":1,"contain":1,"method":1,"construct":1,"execut":2},"maxFreq":7,"totalTerms":27},"loc":3}